src/main/java/com/project/service/CustomUserDetailsService.java

```diff
@@ -19,6 +19,6 @@ public CustomUserDetailsService(UserRepository users) {
 
     @Override
     public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {              // email is used as username
```

Comment:
If this comment is necessary please move it above the `return` statement.

Reply:
fixed